test/inductor/test_select_algorithm.py

```diff
@@ -217,6 +217,36 @@ def foo(a):
         foo(torch.randn(32, 32, device="cuda"))
         self.check_counter(counters["inductor"]["select_algorithm_autotune"], 1)
 
+    @patch.object(select_algorithm, "VERIFY", dict(atol=5e-2, rtol=5e-2))
+    @patches
+    def test_mm_bf16_flags(self):
+        torch.backends.cuda.matmul.flags.allow_bf16_reduced_precision_reduction=True
```

Comment:
Move this to a `patch.object` at decorator so it gets cleaned up at the end.

Reply:
continuing here https://github.com/pytorch/pytorch/pull/108982/files#r1320786697 

This branch is a little off the mark, hope it's ok to take over @michiboo 

I noticed that you weren't even running tests locally.

Reply:
@jon-chuang can you at least leave my commit on, cause I spend some time on this and as a zero time contributor, I would say it is not a good way to welcome a new member to the community.

Reply:
if you really want you can clone the changes in my branch. 

It's just that your tests didn't run, was full of bugs, and the code change itself was one line, whereas it needed to touch a lot more files.

Furthermore, the tests weren't running on CI, so it was at risk of getting merged mistakenly as being correct.

I fixed enough issues that I think it warrants using my branch instead. I recommend trying to at least verify that your changes pass locally for future attempts.

Reply:
@jon-chuang First of all, this issue is labelled as "good first issue" that is why I choose to address it as first time contributor.
As such it is normal to have bugs for first time contributor, I think a little bit of patience will make the world a better place. If issue is more urgent then why labelled it as good first issue? 
I can see you are passionate about this issue, but next time can you please give some time for first time contributor on "beginner issue" and instead of creating separate branch on the issue, you can guide them on their mistakes and bugs.
I think that's more align to the open source values.

Reply:
Well, apologies for my rashness. 

Feel free to merge changes from my branch into your branch, I was just rather alarmed by the number of errors and wanted to make sure it was all fixable.

Thankfully, the tests do indeed fail: https://github.com/pytorch/pytorch/actions/runs/6136917114/job/16654336560?pr=108637 so at least it is caught by CI.